lib/unleash/client.rb

```diff
@@ -11,14 +12,17 @@ module Unleash
   class Client
     attr_accessor :fetcher_scheduled_executor, :metrics_scheduled_executor
 
+    # rubocop:disable Metrics/AbcSize
```

Comment:
Belongs in the Rubocop config but we probably can't get away from this one. It's the only warning generated by this PR but this initialize is literally already at the limit